src/module.rs

```diff
@@ -781,7 +792,8 @@ mod tests {
 		);
 		ModuleInstance::new(
 			&module_with_start,
-			&ImportsBuilder::default()
+			&ImportsBuilder::default(),
+			&InterpreterConfig::default()
```

Comment:
trailing comma missing for this and other cases

Reply:
Sure, didn't know that. will update.